services/spar/test-integration/Test/Spar/DataSpec.hs

```diff
@@ -118,6 +117,19 @@ spec = do
           muid <- runSparCass $ Data.getSAMLUser uref
           liftIO $ muid `shouldBe` Just uid'
 
+      describe "DELETE" $ do
+        it "works" $ do
+          uref <- nextUserRef
+          uid  <- nextWireId
+          do
+            ()   <- runSparCass $ Data.insertSAMLUser uref uid
+            muid <- runSparCass (Data.getSAMLUser uref) `eventually` isJust
+            liftIO $ muid `shouldBe` Just uid
+          do
+            ()   <- runSparCass $ Data.deleteSAMLUser uref
+            muid <- runSparCass (Data.getSAMLUser uref) `eventually` isNothing
+            liftIO $ muid `shouldBe` Nothing
```

Comment:
`shouldBe` after `eventually` seems to contribute nothing in this case. (Unlike `` `shouldBe` Just uid ``.)

Reply:
Oh wait, so `eventually` can return even when `good` has not been achieved?

That's unexpected and I think somebody will bump into this later on.

Reply:
Rename it to `aFewTimes`, and made haddocks more explicit.